python/cugraph/cugraph/link_prediction/wjaccard.py

```diff
@@ -95,47 +112,9 @@ def jaccard_w(input_graph, weights, vertex_pair=None, do_expensive_check=True):
     >>> df = cugraph.jaccard_w(G, weights)
 
     """
-    if do_expensive_check:
-        if not input_graph.renumbered:
-            input_df = input_graph.edgelist.edgelist_df[["src", "dst"]]
-            max_vertex = input_df.max().max()
-            expected_nodes = cudf.Series(range(0, max_vertex + 1, 1)).astype(
-                input_df.dtypes[0]
-            )
-            nodes = (
-                cudf.concat([input_df["src"], input_df["dst"]])
-                .unique()
-                .sort_values()
-                .reset_index(drop=True)
-            )
-            if not expected_nodes.equals(nodes):
-                raise ValueError("Unrenumbered vertices are not supported.")
-
-    if type(input_graph) is not Graph:
-        raise TypeError("input graph must a Graph")
-
-    if type(vertex_pair) == cudf.DataFrame:
-        vertex_pair = renumber_vertex_pair(input_graph, vertex_pair)
-    elif vertex_pair is not None:
-        raise ValueError("vertex_pair must be a cudf dataframe")
-
-    if input_graph.renumbered:
-        # The 'vertex' column of the cudf 'weights' also needs to be renumbered
-        # if the graph was renumbered
-        vertex_size = input_graph.vertex_column_size()
-        # single-column vertices i.e only one src and dst columns
-        if vertex_size == 1:
-            weights = input_graph.add_internal_vertex_id(weights, "vertex", "vertex")
-        # multi-column vertices i.e more than one src and dst columns
-        else:
-            cols = weights.columns[:vertex_size].to_list()
-            weights = input_graph.add_internal_vertex_id(weights, "vertex", cols)
-
-    jaccard_weights = weights["weight"]
-    df = jaccard_wrapper.jaccard(input_graph, jaccard_weights, vertex_pair)
-
-    if input_graph.renumbered:
-        df = input_graph.unrenumber(df, "first")
-        df = input_graph.unrenumber(df, "second")
-
-    return df
+    warning_msg = (
+        "jaccard_w is deprecated. To compute weighted jaccard, please use "
+        "jaccard(input_graph, vertex_pair=False, use_weight=True)"
+    )
```

Comment:
FYI this is a good deprecation message IMO since it clearly directs the user how to change their code.